dashboard-new/src/app/(authenticated)/workflow/[...workflow]/components/Mutations.tsx

```diff
@@ -0,0 +1,57 @@
+import { VariableMutation } from 'littlehorse-client/dist/proto/common_wfspec'
+import { VariableValue } from 'littlehorse-client/dist/proto/variable'
+import { ThreadSpec } from 'littlehorse-client/dist/proto/wf_spec'
+import React, { FC, useMemo } from 'react'
+
+type Props = Pick<ThreadSpec, 'nodes'>
+export const Mutations: FC<Props> = ({ nodes }) => {
+  const mutations = useMemo(() => {
+    return Object.values(nodes).reduce<VariableMutation[]>((acc, node) => {
+      for (const mutation of node.outgoingEdges) {
```

Comment:
We should have a way to show _which outgoingEdge_ the mutation belongs to. The mutations don't belong to the `Node`; rather, they belong to each `edge`. Here is a `WfSpec` where this is important:

```
wf.doIfElse(wf.condition(foo, Comparator.LESS_THAN, bar), ifBody -> {
  ifBody.mutate(myOtherVar, VariableMutationType.ASSIGN, true);
}, elseBody -> {
  elseBody.mutate(myOtherVar, VariableMutationType.ASSIGN, valse);
});
```